mtags/src/main/scala/scala/meta/internal/pc/AutoImports.scala

```diff
@@ -48,13 +59,14 @@ trait AutoImports { this: MetalsGlobal =>
           case Some(pkg)
               if pkg.symbol != rootMirror.EmptyPackage ||
                 pkg.stats.headOption.exists(_.isInstanceOf[Import]) =>
-            val lastImport = pkg.stats
-              .takeWhile(_.isInstanceOf[Import])
+            val lastImportLine = pkg.stats
+              .collect { case imp: Import => imp }
```

Comment:
This changes the behavior in the following case
```
package a

import b.c

class A { import Foo@@ }

import d.e
```

We want to use `takeWhile` so that we stop at `import b.c` instead of `import d.e`